.github/workflows/lint-and-test.yml

```diff
@@ -0,0 +1,45 @@
+## Reference: https://github.com/helm/chart-testing-action
+---
+name: Linting and Testing
+on: pull_request
+jobs: 
+  chart-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+
+      - name: Set up python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Setup Chart Linting
+        id: lint
+        uses: helm/chart-testing-action@v2.0.1
+
+      - name: List changed charts
+        id: list-changed
+        run: |
+          ## If executed with debug this won't work anymore.
+          changed=$(ct --config ./ct.yaml list-changed)
+          charts=$(echo "$changed" | tr '\n' ' ' | xargs)
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
+            echo "::set-output name=changed_charts::$charts"
+          fi
+      - name: Run chart-testing (lint)
+        run: ct lint --debug --config ./ct.yaml --lint-conf ./lintconf.yaml
```

Comment:
Can we move both the `ct.yaml` and the `lintconf.yaml` to the `.github` directory? I don't like having these files in the root directory :-)